docs/book/products/search.rst

```diff
@@ -0,0 +1,61 @@
+.. index::
+   single: Search
+
+Search
+======
+
+Having a products search functionality in an ecommerce system is a very popular usecase. Sylius provides a products search functionality
+that is a grid filter.
+
+Grid filters
+------------
+
+For simple usecases of products search use the **filters of grids**.
+The products grid in the shop, for echa category of products has already a ``search`` filter:
```

Comment:
`echa` -> `each`

Maybe instead:

For example, the shop's categories each have a `search` filter in the products grid: